django/db/models/sql/compiler.py

```diff
@@ -907,6 +907,8 @@ def find_ordering_name(
 
             results = []
             for item in opts.ordering:
+                if hasattr(item, "prefix_references"):
```

Comment:
I'd suggest moving the `item.prefix_references` part to the `if isinstance(item, Orderby)` instead as all expressions are already wrapped in an `OrderBy` and the latter always has a `prefix_references` method so this conditional is unnecessary

```python
if isinstance(item, OrderBy):
    results.append((item.prefix_references(f"{name}{LOOKUP_SEP}"), False))
    continue
```